CMakeLists.txt

```diff
@@ -67,6 +67,11 @@ option(FIREBASE_CPP_BUILD_PACKAGE
 option(FIREBASE_CPP_USE_PRIOR_GRADLE_BUILD
         "When building with Gradle, use the previously built libraries." OFF)
 
+set(FIREBASE_ANDROID_STL "" CACHE STRING "STL implementation to use.")
+if (NOT FIREBASE_ANDROID_STL STREQUAL "")
+  set(ANDROID_STL ${FIREBASE_ANDROID_STL})
```

Comment:
When I tried setting ANDROID_STL on the cmake command line it didn't pick it up for some reason. Rather than exploring why I just went ahead with this workaround. :)

Reply:
Oh ok. I just ran a quick test and see the value being printed out, unless something else later down the line changes this value.

```
message(STATUS "ANDROID_STL ${ANDROID_STL}")
set(FIREBASE_ANDROID_STL "" CACHE STRING "STL implementation to use.")
if (NOT FIREBASE_ANDROID_STL STREQUAL "")
  set(ANDROID_STL ${FIREBASE_ANDROID_STL})
endif()

$cmake -S . -B build_test -DANDROID_STL=c++_static
-- ANDROID_STL c++_static
-- The C compiler identification is AppleClang 11.0.3.11030032
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
....
```